CMakeLists.txt

```diff
@@ -231,6 +232,47 @@ target_link_libraries(test_db_grpc PUBLIC
 #     mcap::mcap
 # )
 
+
+# MCAP REPLAY
+set(REPLAY_DIR ${CMAKE_CURRENT_SOURCE_DIR}/test/test_mcap_replay)
```

Comment:
why do this? no need to have a variable here

Reply:
Did it because it makes sense to me to not repeat the same root dir a bunch of times, but can be changed if you veto it.